docs/installation.rst

```diff
@@ -2,34 +2,42 @@
 Installation
 ============
 
-The recommended approach for installing Planemo is to use Homebrew_ or
-linuxbrew_. To install planemo this way simply use the ``brew`` command as
-follows.
+The recommended approach for installing Planemo is to use Conda_
```

Comment:
Probably yes - That conda recipe doesn't get updated nearly as frequently as the Homebrew or PyPI recipe yet. I'm still nervous about using conda environments with Galaxy also - though I guess it should be fine.

I'll try to adjust my workflow with the GCC as impetus and hopefully we can reach a point where that conda recipe is my go to one.